gossip/src/duplicate_shred_handler.rs

```diff
@@ -40,10 +43,12 @@ pub struct DuplicateShredHandler {
     blockstore: Arc<Blockstore>,
     leader_schedule_cache: Arc<LeaderScheduleCache>,
     bank_forks: Arc<RwLock<BankForks>>,
+    epoch_schedule: EpochSchedule,
```

Comment:
With the new feature activation check code, I don't think we need to cache `epoch_schedule` anymore and we can revert this and `cached_slots_in_epoch`.